Sources/scipio/CreateCommands.swift

```diff
@@ -41,3 +53,15 @@ extension Scipio {
         }
     }
 }
+
+private let availablePlatforms: [SDK] = [.iOS, .macOS, .tvOS, .watchOS]
```

Comment:
how about make this type as "Set"?

Reply:
It should have order for readability. Use OrderedSet instead.